src/Campaigns/ServiceProvider.php

```diff
@@ -34,6 +34,9 @@ public function boot(): void
         $this->registerMenus();
         $this->registerActions();
         $this->registerMigrations();
+      
```

Comment:
Let's put these in the `registerActions` method - since it already exists.

Reply:
I extracted a new `setupCampaignPages()` method, see [a0abd60](https://github.com/impress-org/givewp/pull/7510/commits/a0abd6061d5c29950a9bf0f95fb46f3be0407f21).